content/graphql/field-middleware.md

```diff
@@ -43,6 +43,8 @@ Now whenever we request the `title` field of `Recipe` object type, the original
 
 > info **Hint** To learn how you can implement a field-level permissions system with the use of [extensions](/graphql/extensions) feature, check out this [section](/graphql/extensions#using-custom-metadata).
 
+> warning **Warning** Field middleware can be applied ObjectType only. For more details, check out this [issue](https://github.com/nestjs/graphql/issues/2446).
```

Comment:
```suggestion
> warning **Warning** Field middleware can be applied only to `ObjectType` classes. For more details, check out this [issue](https://github.com/nestjs/graphql/issues/2446).
```

Reply:
Sorry, I missed that. Thanks for updating and reviewing !!